blockly-core/core/ui/block_space/trashcan.js

```diff
@@ -86,7 +86,7 @@ Blockly.Trashcan.prototype.MARGIN_SIDE_ = 22;
 * @type {number}
 * @private
 */
-Blockly.Trashcan.prototype.MARGIN_HOTSPOT_ = 25;
+Blockly.Trashcan.MARGIN_HOTSPOT_ = 25;
```

Comment:
What's the argument for having these on the Trashcan object instead of as local vars? It also seems a little weird/unnecessary to mark a constant (all caps) as private. (This is mostly just me musing - not real need to change).